src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -501,100 +499,100 @@ struct PROBLEME_HEBDO
     uint year = 0;
 
     /* Business problem */
-    char OptimisationAuPasHebdomadaire;
-    char TypeDeLissageHydraulique;
-    char WaterValueAccurate; /* OUI_ANTARES /NON_ANTARES*/
-    char OptimisationAvecCoutsDeDemarrage;
-    int NombreDePays;
-    const char** NomsDesPays;
-    int NombreDePaliersThermiques;
+    char OptimisationAuPasHebdomadaire = NON_ANTARES;
+    char TypeDeLissageHydraulique = PAS_DE_LISSAGE_HYDRAULIQUE;
+    char WaterValueAccurate = NON_ANTARES;
+    char OptimisationAvecCoutsDeDemarrage = NON_ANTARES;
+    int NombreDePays = 0;
+    const char** NomsDesPays = nullptr;
+    int NombreDePaliersThermiques = 0;
 
-    int NombreDInterconnexions;
-    int* PaysOrigineDeLInterconnexion;
-    int* PaysExtremiteDeLInterconnexion;
-    COUTS_DE_TRANSPORT** CoutDeTransport;
+    int NombreDInterconnexions = 0;
+    int* PaysOrigineDeLInterconnexion = nullptr;
+    int* PaysExtremiteDeLInterconnexion = nullptr;
+    COUTS_DE_TRANSPORT** CoutDeTransport = nullptr;
 
-    VALEURS_DE_NTC_ET_RESISTANCES** ValeursDeNTC;
-    VALEURS_DE_NTC_ET_RESISTANCES** ValeursDeNTCRef;
+    VALEURS_DE_NTC_ET_RESISTANCES** ValeursDeNTC = nullptr;
+    VALEURS_DE_NTC_ET_RESISTANCES** ValeursDeNTCRef = nullptr;
 
-    int NombreDePasDeTemps;
-    int NombreDePasDeTempsRef;
-    int* NumeroDeJourDuPasDeTemps;
+    int NombreDePasDeTemps = 0;
+    int NombreDePasDeTempsRef = 0;
+    int* NumeroDeJourDuPasDeTemps = 0;
```

Comment:
Not sure if these initializations really make sense. TBD.

Reply:
Better safe than sorry